lib/spectcl.js

```diff
@@ -476,13 +482,35 @@ Spectcl.prototype.expect = function(expArr, cb){
  * @return {undefined}
  */
 Spectcl.prototype.send = function(data, cb){
-    debug('[send] writing %s to child stdin',data)
-    this.child.stdin.write(data, function(){
-        debug('[send] wrote to child stdin')
+    var self = this
+    debug('[send] writing %s to child stdin',util.inspect(data))
+
+    var writeErrListener = function(err){
+        debug('[send] err %s',err)
+    }
+    self.child.stdin.on('error', writeErrListener)
+
+    try {
```

Comment:
I hate the try wrapping an async call, but I suppose this is the best way to do it...

Can we check if there's a bug in node here? I would like to get this fixed upstream.